Sources/TuistKit/Commands/Signing/EncryptCommand.swift

```diff
@@ -28,12 +29,18 @@ class EncryptCommand: NSObject, Command {
                                      kind: String.self,
                                      usage: "The path to the folder containing the certificates you would like to encrypt",
                                      completion: .filename)
+        keepFilesArgument = subParser.add(option: "--keep-files",
+                                          shortName: "-k",
+                                          kind: Bool.self,
+                                          usage: "Should keep unencrypted files after encryption",
+                                          completion: nil)
         self.signingCipher = signingCipher
     }
 
     func run(with arguments: ArgumentParser.Result) throws {
         let path = self.path(arguments: arguments)
-        try signingCipher.encryptSigning(at: path)
+        let keepFiles = arguments.get(keepFilesArgument) ?? false
```

Comment:
I'd argue that this should be the inverse, by default I would not expect my files to be deleted. Typically these files should be put into `.gitignore` anyway so would not get committed. 

What are your thoughts on this one?

Reply:
Truth be told, I don't have a strong opinion on what should be the default. @pepibumur - what's your take? I added it as a reaction to your [insight](https://github.com/tuist/tuist/pull/1127#issuecomment-606791533):
> What about deleting unencrypted files after encrypting them? I'd default to that and give developers the option to specify whether they don't want those files to be deleted: tuist signing encrypt --keep-files. I think it's good to default to deletion for safety reasons.

Reply:
I'd delete them, even if they are part of the `.gitignore`. Not just for security reasons, because one could modify the `.gitignore` and end up with those files in the git history, but to keep directories tidy. Why would you want to keep a file around that is not necessary? The certificate is in your keychain, and the profile in your developer account. The only thing that the project needs to work is the encrypted version of them.

I'd get rid of the `--keep-files` argument. We _(developers)_ have the tendency of making everything configurable, and we know that that often leads to complexity. Let's remain simple now that we can to provide streamlined experiences to developers.